wolfSSL/src/chapter02.md

```diff
@@ -394,6 +394,122 @@ Removes code for OAEP padding.
 
 Turns off AES-CBC algorithm support.
 
+#### WOLFCRYPT_ONLY
+
+Enables wolfCrypt only while disabling TLS.
+
+#### NO_CAMELLIA_CBC
+
+Disables Camellia CBC support.
+
+#### NO_AES
+
+Disables AES algorithm support.
+
+#### NO_AESGCM_AEAD
+
+Disables AES GCM algorithm support.
+
+#### NO_ASN_TIME
+
+Disables time checking for ASN.
+
+#### NO_CHECK_PRIVATE_KEY
+
+RSA - needed to encrypt salt.
+
+#### NO_DH
+
+Disables DH support.
+
+#### NO_ED25519_CLIENT_AUTH
+
+Disables client authEd25519 requires caching enabled for tracking message hash used in EdDSA_Update for signing.
+
+#### NO_ED448_CLIENT_AUTH
+
+Disables client authentification for ED448.
+
+#### NO_FORCE_SCR_SAME_SUITE
+
+Applys a restriction on secure negotation. Don't allocate Suites' object on renegotiation.
+
+#### NO_MULTIBYTE_PRINT
+
+Used for compiling out special characters that embedded devices may have problems with.
+
+#### NO_OLD_SSL_NAMES
+
+Removes unneeded namespace.
+
+#### NO_OLD_WC_NAMES
+
+Removes unneeded namespace.
+
+#### NO_OLD_POLY1305
+
+For compiling out special characters that embedded devices may have problems with.
+
+#### NO_HANDSHAKE_DONE_CB
+
+Applys a restriction on secure negotation. Don't allocate Suites' object on renegotiation.
+
+#### NO_STDIO_FILESYSTEM
+
+ASN Library Selection (default to ASN_TEMPLATE).
+
+#### NO_TLS_DH
+
+Excludes TLS DH. Should not negotiate cipher suites based on ephemeral finite-field Diffie-Hellman key agreement.
+
+#### NO_WOLFSSL_CM_VERIFY
+
+Sets certification manager callback.
+
+#### NO_WOLFSSL_DIR
+
+Disable directory support.
+
+#### NO_WOLFSSL_RENESAS_TSIP_TLS_SESSION
+
+For disabling only the TSIP TLS-linked Common key encryption method.
+
+#### NO_WOLFSSL_SHA256
+
+Disables support for SHA-256.
+
+#### WOLFSSL_BLIND_PRIVATE_KEY
+
+Used as a mask of private key DER.
+
+#### WOLFSSL_DTLS13_NO_HRR_ON_RESUME
+
+If defined, a DTLS server will not do a cookie exchange on successful client resumption: the resumption will be faster (one RTT less) and will consume less bandwidth (one ClientHello and one HelloVerifyRequest/HelloRetryRequest less). On the other hand, if a valid SessionID/ticket/psk is collected, forged clientHello messages will consume resources on the server. For DTLS 1.3, using this option also allows for the server to process Early Data/0-RTT Data. Without this, the Early Data would be dropped since the server doesn't enter stateful processing until receiving a verified ClientHello with the cookie. To allow DTLS 1.3 resumption without the cookie exchange:- Compile wolfSSL with WOLFSSL_DTLS13_NO_HRR_ON_RESUME defined - Call wolfSSL_dtls13_no_hrr_on_resume(ssl, 1) on the WOLFSSL object to disable the cookie exchange on resumption - Continue like with a normal connection.
+
+#### WOLFSSL_NO_CLIENT_AUTH
+
+Disable client authentification for Ed25519/Ed448.
+
+#### WOLFSSL_NO_CURRDIR
+
+Disable settings used when enabling Arduino.
```

Comment:
This is not specific to Arduino. It is a portability macro for platforms that do not support `./` for test paths in wolfssl/test.h. Applies to testing tools only.